OADP-5261: Update the OADP introduction to include entire cluster backup unsupported (#85311)
openshift/openshift-docs: backup_and_restore/application_backup_and_restore/oadp-intro.adoc

toc::[]

The {oadp-first} product safeguards customer applications on {product-title}. It offers comprehensive disaster recovery protection, covering {product-title} applications, application-related cluster resources, persistent volumes, and internal images. {oadp-short} is also capable of backing up both containerized applications and virtual machines (VMs).

However, {oadp-short} does not serve as a disaster recovery solution for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd] or {OCP-short} Operators.

> {oadp-short} does not support full cluster backup and restore either.

Reviewer: ```suggestion
{oadp-short} support is provided to customer workload namespaces only. Full cluster backup and restore are not supported.
```

> {oadp-short} support is provided to customer workload namespaces only. Full cluster backup and restore are not supported.

Reviewer 2: maybe support is provided to customer workload namespaces, and cluster scope resources. support is provided to customer workload namespaces 
maybe link to the cluster backup and restore from here.

Author: @amastbau - please review again, most wise and powerful one!

toc::[]

The {oadp-first} product safeguards customer applications on {product-title}. It offers comprehensive disaster recovery protection, covering {product-title} applications, application-related cluster resources, persistent volumes, and internal images. {oadp-short} is also capable of backing up both containerized applications and virtual machines (VMs).

However, {oadp-short} does not serve as a disaster recovery solution for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd] or {OCP-short} Operators.

> Nor does {oadp-short} support full cluster backup and restore.

Author: ```suggestion
{oadp-short} does not support full cluster backup and restore either.
```

> 

Author: ```suggestion
```
